### ISSUE TITLE
[a11y] 9.1.3.1a - HTML-Structure elements for headings

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#5377]
- Context: The Web UI should be accessible according to the WCAG2.1 /BITV 2.0 guidelines. The number in the title relates to the test-steps: https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/pruefschritte.html
- "Status" relates to the evaluation scheme https://www.bitvtest.de/bitv_test/das_testverfahren_im_detail/verfahren.html#c1262 
- "a11y" is short for accessibility
- ----------------

## General notes
Note: "Login“-Page (not part of the test): The error message should not be implemented as a h6 heading. Please remove headings markup.

## Status: fulfilled

**Comment by pascalwengerter** [comment]
`Login.js` in `idp/` in `oCIS` needs a `{hasError && <Typography id="oc-login-error-message" variant="subtitle2" *component="p"* color="error"`, otherwise the component type gets inherited from somewhere and it ends up being an (incorrect) `<h6>`
